napari/layers/base/base.py

```diff
@@ -1172,11 +1185,11 @@ def _get_value(self, position):
 
     def get_value(
         self,
-        position: Tuple[float, ...],
+        position: npt.ArrayLike,
```

Comment:
I have mixed feelings. Previous annotations inform that it is single-dimensional. 
```suggestion
        position: Sequence[float],
```

Reply:
hmm I still think npt.ArrayLike fits better as it is described as any `Sequence` that can be converted an array of one `dtype`. But I do understand your point of losing information. I do think it is clear from the argument name though and the function definition.

Reply:
Yes. but for `Sequence[float]` valid is `(1, 1)` 

For `npt.ArrayLike` valid is also `[[1, 1],[1,1]]`

Reply:
Yeah I agree with @Czaki here — until we can use [variadic generics](https://peps.python.org/pep-0646/) with npt.ArrayLike, `Sequence[float]` is a better fit imho.

Reply:
(And I *think* that means, until Py3.11 is our minimum supported version, but I don't know whether PEP-646 was backported to earlier versions.)

Reply:
ok you convinced me @Czaki. @dstansby can you implement the change please?:)

Reply:
I started making this change, but `Sequence[float]` doesn't alow for something of type `np.ndarray` to be passed in, which we definitely want to allow.

I do agree that where possible we want to specify the number of dimensions, but in the case of positions I decided to go with `ArrayLike` because we want to always cast the input to an array to do array operations (e.g. multiplication, division, norming) on the position.

So for the purpose of keeping the typing simple, I'd advocate for staying with `ArrayLike` instead of `Sequence` - thoughts?

Reply:
The only other option I would say is a `Union` of `ArrayLike` and `Sequence`. Or to add the docstring in which it is stated that single dimension is expected.

Reply:
@jni @Czaki Personally I would say the latter option keeping `ArrayLike` as type hint, but raising `ValueError` if not 1-dimensional is the way to go. What do you think?

Reply:
Wow I'm quite devastated that ndarray doesn't pass typing as a Sequence[float], but I guess it makes sense. I'm not opposed to that approach then. Union is too clunky.